pkg/uploader/kopia/block_backup_nonwindows.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
```

Comment:
If you are using buildtags I don't think you need to rename the file?

Reply:
Modified.